package.json

```diff
@@ -9,6 +9,16 @@
     "front-end",
     "framework"
   ],
+  "scripts": {
+    "dev": "gulp",
+    "build": "gulp build",
+    "watch": "gulp watch",
+    "lint": "gulp lint"
+  },
+	"engines": {
+		"node": "14",
+		"npm": "8"
+	},
```

Comment:
Is this really needed? If yes please update to the LTS versions as well.
Also: should be indented using spaces.